scripts/db_migrations.py

```diff
@@ -92,19 +92,33 @@ def moveAuthInformationToProperties(self, resources_container_name):
 
         for item in resources_container.query_items(query='SELECT * FROM c', enable_cross_partition_query=True):
             template_version = semantic_version.Version(item["templateVersion"])
-            if (template_version < semantic_version.Version('2.5.0') and "authInformation" in item):
-                print(f'Found workspace {item["id"]} that needs migrating')
+            updated = False
+            if (template_version < semantic_version.Version('0.2.9') and item["templateName"] == "tre-workspace-base"):
 
                 # Rename app_id to be client_id
-                item["properties"]["client_id"] = item["properties"]["app_id"]
-                del item["properties"]["app_id"]
-                del item["authInformation"]["app_id"]
+                if "app_id" in item["properties"]:
+                    item["properties"]["client_id"] = item["properties"]["app_id"]
+                    del item["properties"]["app_id"]
+                    updated = True
 
-                # merge authInformation into properties
-                item["properties"] = {**item["authInformation"], **item["properties"]}
-                del item["authInformation"]
-                resources_container.upsert_item(item)
-                print(f'Upgraded authentication info for workspace id {item["id"]}')
+                if "scope_id" not in item["properties"]:
+                    updated = True
```

Comment:
Very nit: you put `updated = True` as a last statement in other conditions, could you do the same here for consistency